lib/ethdev/rte_ethdev_netns.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <fcntl.h>
+#include <unistd.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdint.h>
+
+#include "rte_ethdev.h"
+
+inline int rte_ethdev_enter_netns(int ns)
+{
+	int origNs = open("/proc/thread-self/ns/net", O_RDONLY | O_CLOEXEC);
+	if (origNs >= 0) {
+		setns(ns,0);
+	}
+	printf("Setez un ns, destinatia %i curent %i\n",ns, origNs);
+	return origNs;
+}
+
+inline int rte_ethdev_switch_netns(char * netns, int * deviceNS, int * originalNS)
+{
+	char ns[128];
+
+	if (netns[0] == '\0')
+		return -1;
+
+	snprintf(ns, 128, "/var/run/netns/%s", netns);
+	*deviceNS = open(ns, O_RDONLY | O_CLOEXEC);
```

Comment:
what happens if this open fails?

Reply:
Redid this checking to return an error when the `open` fails